client/js/store/core/initStore.js

```diff
@@ -72,16 +72,22 @@ function initStore (storeModules, apiStoreModules, presetStoreModules) {
             },
             successCallbacks: [
               async (success) => {
-                const response = await success.json()
+                // If the response body is empty, contentType will be null
```

Comment:
please adapt these changes for the error-case too.

Reply:
Done in d5c798cda19c1efa26c0d832acb88cf257fbdd8e